opensrp-family/src/main/java/org/smartregister/family/domain/FamilyMetadata.java

```diff
@@ -59,7 +59,7 @@ public void updateFamilyOtherMemberRegister(String tableName, int currentLimit,
         this.familyOtherMemberRegister = new FamilyOtherMemberRegister(tableName, currentLimit, showPagination);
     }
 
-    public class FamilyRegister {
+    public static class FamilyRegister {
```

Comment:
what is the reason for this change

Reply:
Changing `FamilyRegister` from an inner class to a nested class `FamilyRegister`. 
Should allow creating an instance of FamilyRegister outside the scope of `FamilyMedata` without breaking underlying functionality